packages/@angular/cli/commands/generate.ts

```diff
@@ -64,8 +64,11 @@ export default Command.extend({
   ],
 
   getCollectionName(rawArgs: string[], parsedOptions?: { collection?: string }): [string, string] {
-    let schematicName = rawArgs[0];
     let collectionName = CliConfig.getValue('defaults.schematics.collection');
+    if (!rawArgs) {
+      return collectionName;
```

Comment:
The function seems to expect an array to be returned and from what I can tell, this will be just a single string returned.